frontend/main/src/pages/course/[coursePath]/index.tsx

```diff
@@ -25,14 +30,109 @@ export default function Post({
+              {post.coverPhoto?.path ? (
+                <>
+                  <Image
+                    src={post.coverPhoto?.path}
+                    alt={post.title}
+                    width="1920"
+                    height="1080"
+                    layout="responsive"
+                    className=""
+                  />
+                </>
+              ) : (
+                <div className="flex items-center flex-auto rounded-t-md bg-primary-900 dark:bg-primary-900">
+                  <AJPrimary className="max-w-full p-4 mx-auto max-h-32 2xl:max-h-64" />
+                </div>
+              )}
+              {/* Beginner/Intermediate/Advanced descriptor */}
+              <p className="p-2 rounded-full text-basics-50 dark:text-basics-50 bg-secondary-600 dark:bg-secondary-600">
+                Beginner
+              </p>
+              <div className="flex items-center justify-center space-x-4 flex-nowrap">
+                <button className="btn-secondary">Buy Now</button>
+                <button className="btn-primary">Become a Member</button>
+              </div>
+            </section>
+          )}
+          <section className="grid content-center p-4 pt-6 bg-secondary-600 dark:bg-secondary-600 text-basics-50 dark:text-basics-50">
+            {/* Title */}
+            <div className="grid grid-cols-1 gap-4 mx-auto">
+              <h1 className="text-4xl lg:text-7xl">{post.title}</h1>
+              {/* Course Description */}
+              <p className="max-w-sm">{post.content}</p>
+              {/* Instructor */}
+              <div className="p-2 xl:p-4 bg-secondary-600 rounded-t-md dark:bg-secondary-600">
+                <div className="flex gap-4 ">
+                  <img
+                    src="https://avatars0.githubusercontent.com/u/45889730?s=460&u=74587a01abf2a7f33ae964c69856f3fe71b175b6&v=4"
+                    alt="instructor"
+                    className="w-20 h-20 border-2 rounded-full border-primary-900"
+                  />
+
+                  <div className="flex flex-col justify-center">
+                    <h3 className="m-0 text-base font-light">Instructor</h3>
+                    <h4 className="m-0 text-xl">Instructor Name</h4>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            {/* Add Instructor Page with List of Instructors and their Bios */}
+            {/* <p className="p-2 xl:p-4">
+          Instructor description: Lorem ipsum dolor sit amet consectetur
+          adipisicing elit. Sint ad iusto nobis excepturi deserunt
+          exercitationem ex aspernatur sit culpa fugit porro, facere eaque.
+          Harum consequuntur corrupti odio blanditiis, culpa officia!
+        </p> */}
+          </section>
+        </section>
+      </section>
+
+      {post.sections?.map((section) => {
```

Comment:
This will need to look more like this

```
      {post.sections &&
        post.sections.map((section, i) => {
          return (
            <section key={i}>
              <h3>{section.title}</h3>
              <ul>
                {section.lessons?.map((lesson, i) => {
                  return (
                    <li key={i}>
                      <h4>{lesson.title}</h4>
                    </li>
                  );
                })}
              </ul>
            </section>
          );
        })}
```